### DISCUSSION TITLE
"All tuners already in use"

### DISCUSSION THREAD

**Discussion by ghost** [tgorgdotcom/locast2plex#227] [Q&A]
On my Plex Media Server I am getting the following error in the PMS log when trying to record a show:

`Feb 26, 2021 16:00:01.129 [0xa9cf7430] ERROR - [Transcoder] http://192.168.2.206:6077/watch/1571514243723: Server returned 400 Bad Request
`
When I hit my locast2plex server with `http://192.168.2.206:6077/` I get the following nice 200 response

```
<MediaContainer>
    <MediaGrabber identifier="tv.plex.grabbers.locast2plex" title="Locast2Plex" protocols="livetv" />
</MediaContainer>

```
However, when I hit the locast2plex server from a **browser** with the request above which returned the 400 I get the following

```
<html>
    <head></head>
    <body>
        <h2>All tuners already in use.</h2>
    </body>
</html>
```
Any thoughts?

**Comment by djsiegel** [comment]
I had this issue this week. All my scheduled recordings in Plex stopped on a certain date. 
I was able to simply restart the locast2plex container and it looks like that freed up the tuners. L2P is able to record in plex again.
